formatTest/unit_tests/expected_output/infix.re

```diff
@@ -891,30 +891,30 @@ let containingObject = {
     /* Same as */
     let result = 2 + (- add 4 0);
     /* That same example but with ppx attributes on the add application */
-    let result = 2 + (- add 4 0 [@ppx]);
+    let result = 2 + (- [@ppx] add 4 0);
     /* Same as */
-    let result = 2 + (- add) 4 0 [@ppx];
+    let result = [@ppx] 2 + (- add) 4 0;
```

Comment:
I can't figure out how this works syntactically -- shouldn't `4 0` be inside the `add` parens?